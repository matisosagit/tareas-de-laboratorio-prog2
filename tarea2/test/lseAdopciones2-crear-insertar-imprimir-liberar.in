crearLSEAdopcionesVacia
imprimirLSEAdopciones
liberarLSEAdopciones
crearLSEAdopcionesVacia
crearFecha 18/08/1930
crearPerro 1 Pluto 4 8 Perrito simpaticon
crearPersona 12345678 Mickey Mouse 18 11 1928
agregarPerroPersona
crearFecha 1/1/2025
insertarLSEAdopciones
imprimirLSEAdopciones
liberarLSEAdopciones
Fin
