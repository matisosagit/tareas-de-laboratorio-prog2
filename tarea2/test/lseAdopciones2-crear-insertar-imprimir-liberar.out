1>2>Lista de adopciones:
Fin lista de adopciones.
3>4>5>6>7>8>9>10>Adopcion de persona de CI 12345678 y perro de id 1 agregada de forma exitosa.
11>Lista de adopciones:
---------------------------
Adopcion en fecha 1/1/2025
Adoptante:
Persona Mickey Mouse
CI: 12345678
Adoptado:
Perro 1
Nombre: Pluto
Fecha de ingreso: 18/8/1930
---------------------------
Fin lista de adopciones.
12>13>Fin.
